src/commands/status.ts

```diff
@@ -1,30 +1,43 @@
-import { Client, Message } from 'discord.js';
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { ActivityType, CommandInteraction } from 'discord.js';
+import { Command } from '../types/command';
+import { CommandBody } from '../types/command-body';
 
-exports.run = async (client: Client, msg: Message, args: string[]) => {
-  if (args === undefined || args.length === 0) {
-    const desc = module.exports.help.description;
-    const name = module.exports.help.name;
-    const usage = module.exports.help.usage;
-    msg.channel.send(`Name: ${name}\nDescription: ${desc}\nUsage: ${usage}`);
-    return;
-  }
+const data = new SlashCommandBuilder()
+  .setName('status')
+  .setDescription('Set Rusty\'s current status.')
+  .addStringOption(option =>
+    option
+      .setName('activitytype')
+      .setDescription('Type of activity that discord will display')
+      .setRequired(true)
+      .addChoice('Playing', 'PLAYING')
+      .addChoice('Listening', 'LISTENING')
+      .addChoice('Streaming', 'STREAMING')
```

Comment:
We should probably verify the streaming choice, last I tested it didn't work without a URL being specified which this command doesn't support.

Reply:
Decided to comment for now, can look into more in https://github.com/Mr1upMachine/rusty-bot/issues/72